src/Chapter1.hs

```diff
@@ -556,7 +574,11 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c
+  | c `elem` vowels = True
+  | otherwise = False
+  where vowels = ['a', 'e', 'i', 'o', 'u']
```

Comment:
Nice one! 👍🏼 
As `elem` returns `Bool` itself, you can just return the result of this function:

```haskell
isVowel c = c `elem` ['a', 'e', 'i', 'o', 'u']
```

One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎

Reply:
Sounds good. Interesting that explicit pattern matching is faster 🤔

Reply:
@cherealnice The future version of GHC actually will implement such optimization to automatically rewrite `elem` with hardcoded list to manual pattern matching. But it was discovered recently, that `elem` is much slower, and rewriting it to pattern matching in one of the Haskell projects significantly improved the performance of application, because such `elem` was called a lot in there.